darts/models/forecasting/transformer_model.py

```diff
@@ -480,7 +480,7 @@ def encode_year(idx):
                 }
             ..
         random_state
-            Control the randomness of the weights initialization. Check this
+            Control the randomness of the weight's initialization. Check this
```

Comment:
Let's revert this change, since there are several weights, the apostrophe does not seem appropriate.

Or we could eventually rephrase it into "Control the randomness in the initialization of the weights" if you have the impression that the original sentence was not clear.